tests/testthat/test-plot_seromodel.R

```diff
@@ -0,0 +1,313 @@
+        geom_params = list(
+          xmin = 0, xmax = 1, ymin = 0.5, ymax = 1,
+          scale = 1, clip = "inherit", halign = 0.5, valign = 0.5
+        )
+      ),
+      list(mapping = NULL, geom_params = list(
+        xmin = 0, xmax = 1,
+        ymin = 0, ymax = 0.5, scale = 1, clip = "inherit", halign = 0.5,
+        valign = 0.5
+      ))
+    ),
+    coordinates = list(
+      limits = list(x = c(
+        0,
+        1
+      ), y = c(0, 1))
+    ),
+    labels = list()
+  )
+
+  expect_lists_equal_with_tolerance(expected_plot, actual_plot)
+})
+
+
+test_that("plot_seroprevalence_estimates creates a ggplot with correct structure", {
+  skip_on_cran()
+
+  seromodel <- fit_seromodel(
+    serosurvey = serosurvey
+  )
+
+
+  plot <- plot_seroprevalence_estimates(
+    seromodel = seromodel,
+    serosurvey = serosurvey
+  )
+  actual_plot <- extract_plot_data(plot)
+
+  expected_plot <- list(
+    classes = c("gg", "ggplot"), layers = list(
+      list(mapping = list(
+        ymin = "seroprev_lower", ymax = "seroprev_upper"
+      ), geom_params = list(
+        na.rm = FALSE, orientation = NA, width = 0.1
+      )), list(mapping = list(
+        y = "seroprev", size = "n_sample"
+      ), geom_params = list(na.rm = FALSE)),
+      list(mapping = list(x = "age", y = "median"), geom_params = list(
+        na.rm = FALSE, orientation = NA
+      )), list(mapping = list(
+        x = "age", ymin = "lower", ymax = "upper"
+      ), geom_params = list(
+        na.rm = FALSE, orientation = NA, outline.type = "both"
+      ))
+    ),
+    coordinates = list(limits = list(x = c(0, 60), y = NULL)),
+    labels = list(
+      x = "Age", y = "Seroprevalence", ymin = "seroprev_lower",
+      ymax = "seroprev_upper", size = "n_sample"
+    )
+  )
+
+  expect_lists_equal_with_tolerance(expected_plot, actual_plot)
+})
+
+test_that("plot_foi_estimates creates a ggplot with correct structure", {
+  skip_on_cran()
+
+  seromodel_age <- fit_seromodel(
+    serosurvey = serosurvey,
+    model_type = "age"
+  )
+
+  plot <- plot_foi_estimates(
+    seromodel = seromodel_age,
+    serosurvey = serosurvey
+  )
+  actual_plot <- extract_plot_data(plot)
+
+  expected_plot <- list(
+    classes = c("gg", "ggplot"), layers = list(list(mapping = list(
+      ymin = "lower", ymax = "upper"
+    ), geom_params = list(
+      na.rm = FALSE,
+      orientation = NA, outline.type = "both"
+    )), list(mapping = list(
+      y = "median"
+    ), geom_params = list(na.rm = FALSE, orientation = NA))),
+    coordinates = list(limits = list(x = NULL, y = c(0, 0.179333524440058))), labels = list(
+      x = "Age", y = "Force-of-Infection", ymin = "lower",
+      ymax = "upper"
+    )
+  )
+
+  expect_lists_equal_with_tolerance(expected_plot, actual_plot)
+})
+
+test_that("plot_rhats creates a ggplot with correct structure", {
+  skip_on_cran()
+
+  seromodel_age <- fit_seromodel(
+    serosurvey = serosurvey,
+    model_type = "age"
+  )
+
+  plot <- plot_rhats(
```

Comment:
I just realized that there is not a export tag for this function. I oppened issue #226 to address this before merging this test to dev.